backend/src/main/java/pl/krkteam/battleships/room/holder/dto/room/list/RoomListDTO.java

```diff
@@ -0,0 +1,18 @@
+package pl.krkteam.battleships.room.holder.dto.room.list;
+
+import java.util.Arrays;
+
+public class RoomListDTO {
+    private RoomDTO[] roomName;
```

Comment:
try if this field can be a list

Reply:
For sure transfered object must be in array (in angular there is no List) and in Java you can use Arrays.asList() or use TypeToken to transfer array to list.